src/servicelevelagreement/logic/logic.ergo

```diff
@@ -39,23 +38,24 @@ define function createPaymentObligation(
       amount: amount,
       description:
         "payment owed by "
-          ++ serviceProvider.partyId
-          ++ " to " ++ serviceConsumer.partyId
+					++ toString(serviceProvider)
```

Comment:
Some weird indentation?

Reply:
Good catch. I really need to figure out tabbing configuration for syntax editing.